This is a patch for PICkit3 scripting firmware (pk3img020000.hex)
to adapt it for Microstick II.  On power on, PICkit keeps /MCLR signal low.
This patch make is high, to allow the target processor to run.
___
2012-07-16
Serge Vakulenko

--- ../pk3img020000.hex	2012-04-19 12:35:50.000000000 -0700
+++ pk3img020000-noreset.hex	2012-07-15 23:49:10.000000000 -0700
@@ -787,7 +787,7 @@
 :10311000FFB89700C508DD0042C0BF007F406000D7
 :103120000140700042E0B700920288000080FA007F
 :103130004F0278004F01BE004F00BE000040060065
-:103140000004E0000E003200E1FE2000001020002C
+:103140000000000000000000E1FE20000010200050
 :10315000110060000700320070402000800C8800E1
 :103160001170A000801A260001002000E611020064
 :1031700000000000E4C2A900E4A2A8000F0037008C
